src/components/merge_results/fields/CompareFundings.tsx

```diff
@@ -0,0 +1,42 @@
+import { Typography } from '@mui/material';
+import { useContext } from 'react';
+import { useFormContext, useWatch } from 'react-hook-form';
+import { useTranslation } from 'react-i18next';
+import { Registration } from '../../../types/registration.types';
+import { MergeResultsWizardContext } from '../MergeResultsWizardContext';
+import { FundingBox } from './FundingBox';
+
+export const CompareFundings = () => {
+  const { t } = useTranslation();
+
+  const { control } = useFormContext<Registration>();
+  const { sourceResult } = useContext(MergeResultsWizardContext);
+  const sourceFundings = sourceResult.fundings ?? [];
+
+  const targetFundings = useWatch({ name: 'fundings', control }) ?? [];
+
+  return (
+    <>
+      <Typography variant="h4" sx={{ display: { xs: 'none', md: 'block' } }}>
+        {t('common.funding')}
+      </Typography>
+      <Typography variant="h4" sx={{ gridColumn: { xs: 1, md: 3 } }}>
+        {t('common.funding')}
+      </Typography>
+
+      <div>
+        {sourceFundings.map((funding) => {
+          return <FundingBox key={`${funding.identifier}-${funding.id}`} funding={funding} />;
```

Comment:
Hvorfor `${funding.identifier}-${funding.id}` som key her, mens target bruker bare `funding.identifier`?
key burde vel vært utformet likt for begge?